JumpscaleCore/data/schema/tests/1_base.py

```diff
@@ -7,6 +7,7 @@ def main(self):
 
     kosmos 'j.data.schema.test(name="base")'
     """
+    j.data.schema.meta.reset()
```

Comment:
this should be `j.data.schema.reset()`

Reply:
you are right and I have updated the code